samza-core/src/main/java/org/apache/samza/config/JavaSystemConfig.java

```diff
@@ -60,4 +66,33 @@ public String getSystemFactory(String name) {
     }
     return systemNames;
   }
+
+  /**
+   * @return map of system name to {@link SystemAdmin}
+   */
+  public Map<String, SystemAdmin> getSystemAdmins() {
```

Comment:
What do you suggest? There's currently no method in the scala SystemConfig to get the factories or the admins, only the names.